index.html

```diff
@@ -1,214 +1,255 @@
+                About me 
+            </h2>
+        </section>
+        <section class="text">
+            <p>
+                I can help you build a product , feature or website Look through some of my work and <br>experience! If you like what you see and have a project you need coded,  <br>don’t hesitate to contact me.
+            </p>
+        </section>
+        <button>
+            Get my resume
+        </button>
+    </div>
+    <div class="me-container">
+        <section class="card">
+            <section id="lang">
+            </section>
+            <h2>Languages</h2>
+            <ul>
+                <li>JavaScript </li>
+                <li>Ruby </li>
+                <li>HTML </li>
+                <li>CSS </li>
+            </ul>
+        </section>
+        <section class="card">
+            <section id="fw">
+            </section>
+            <h2>Frameworks</h2>
+            <ul>
+                <li>React.js</li>
+                <li>Ruby on Rails </li>
+                <li>RSpec </li>
+                <li>Capybara </li>
+                <li>Selenium </li>
+            </ul>
+        </section>
+        <section class="card">
+            <section id="sk">
+            </section>
+            <h2>Skills</h2>
+            <ul>
+                <li>Database Management </li>
+                <li>Version Control </li>
+                <li>CLI </li>
+                <li>Web Development </li>
+                <li>API Design </li>
+            </ul>
+        </section>
+    </div>
+    <div class="form-container">
+        <section class="touch">
+            <h2> 
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h2>
+        </section>
+        <form class="connect" action="https://formspree.io/f/mwkjdnnq" method="POST" autocomplete="off">
+            <input id="name" name="username" placeholder="Full name" required type="text" maxlength="30"/>
+            <input id="mail" name="mail" placeholder="Email adress" required type="email"/>
+            <textarea id="comment" name="comment" maxlength="500" placeholder="Hey I’d like to connect to... " required></textarea>
+            <button>
+                Get in touch
+            </button> 
+        </form>  
+        <section class="mail">
+            <p>ngachaian@gmail.com</p>
+        </section>        
+     </div>
+     <footer class="foot">
+        <section class="socials">
+            <ul>
+                <li> <a href=""> <img src="Images/twitter.png" alt="Twitter"/> </a> </li>
+                <li> <a href=""> <img src="Images/Linkedin icon.png" alt="LinkedIn"/> </a> <li>
+                <li> <a href=""> <img src="Images/medium.png" alt="Medium"/> </a> </li>
+                <li> <a href=""> <img src="Images/github.png" alt="GitHub"/> </a> </li>
+                <li> <a href=""> <img src="Images/angellist.png" alt="Angellist"/> </a> </li>
+            </ul>
+        </section>
```

Comment:
As suggested by the previous reviewer, kindly implement the following changes in your `contact` section to enable your design to have a similar view to the figma template given :

- [ ] The text on the left-hand side doesn't seem to be in the right position and alignment as in the design given, please implement these changes  and also remove the email as it is not supposed to be present there
- [ ] Also, the contact boxes doesn't have the width, padding, and alignment as it is shown in the figma template. Please implement this change to have a better visual representation of your project

Your design :
![image](https://github.com/NgachaIan/Personal_Portfolio/assets/20537712/52b13652-b934-4ca7-b7af-31ee2f959ffa)

Figma design :
![image](https://github.com/NgachaIan/Personal_Portfolio/assets/20537712/1e3d51c5-1055-460e-b9cd-cd663f6d82b2)